packages/file-graph/fileGraph.js

```diff
@@ -1,17 +1,13 @@
 import FileNode from './fileNode.js'
 
-export default class FileGraph {
-  constructor(root) {
-    this.root = new FileNode(root)
-  }
-
+export default class FileGraph extends FileNode {
```

Comment:
How does this line function?

Reply:
It extends the properties and method from FileNode (and the FileNode extends from File as well,) so FileGraph it self is also a FileNode and can use all the methods.